faqs/gtn/loading_data.md

```diff
@@ -0,0 +1,88 @@
+---
+title: How can i load data?
+area: introduction
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+
+
+## Video Tutorials
+
+* Quick help: [**Manupulating NGS data with Galaxy: Getting Data In**](/tutorials/ngs/#getting-ngs-data-in)
+* Full tutorial: [**Uploading data into Galaxy**](/tutorials/upload/)
+* Dataset Collections, including creation during Upload: [**Processing many samples at once with collections**](https://galaxyproject.org/tutorials/collections/)
+* FTP/FTPS tutorial: [**FTP Upload**](/ftp-upload/)
+
+## GTN Tutorials
```

Comment:
I'm not sure this section is an FAQ, and the ones below, shouldn't they be individual FAQs?